packages/block-library/src/audio/edit.native.js

```diff
@@ -55,6 +56,14 @@ function AudioEdit( {
 		setAttributes( { id: mediaId, src: mediaUrl } );
 	};
 
+	const { wasBlockJustInserted } = useSelect( ( select ) => {
+		return {
+			wasBlockJustInserted: select(
+				blockEditorStore
+			).wasBlockJustInserted( clientId, 'inserter_menu' ),
+		};
+	} );
```

Comment:
Nitpick: We could get rid of the `return` line and directly return the object with the values:

```suggestion
	const { wasBlockJustInserted } = useSelect( ( select ) => ( {
		wasBlockJustInserted: select( blockEditorStore ).wasBlockJustInserted(
			clientId,
			'inserter_menu'
		),
	} ) );
```

Reply:
Nice. I will make this change 👍🏾

Reply:
Resolved here b140201